ports/servoshell/egl/ohos.rs

```diff
@@ -513,8 +532,13 @@ impl HostCallbacks {
 
 #[allow(unused)]
 impl HostTrait for HostCallbacks {
-    fn prompt_alert(&self, msg: String, trusted: bool) {
-        warn!("prompt_alert not implemented. Cancelled. {}", msg);
+    fn prompt_alert(&self, msg: String, _trusted: bool) {
+        debug!("prompt_alert: {}", &msg);
+        if let Some(prompt_fn) = PROMPT_TOAST.get() {
+            prompt_fn.call(msg, ThreadsafeFunctionCallMode::Blocking);
+        } else {
+            error!("PROMPT_TOAST not set. Dropping msg {msg}");
+        }
```

Comment:
Since you are handling both cases in the same way, I think a match is slightly more appropriate:

```suggestion
        match PROMPT_TOAST.get() {
            Some(function) => function.call(msg, ThreadsafeFunctionCallMode::Blocking),
            None => error!("PROMPT_TOAST not set. Dropping msg {msg}"),
        }
```